pkgs/games/mindustry/default.nix

```diff
@@ -188,6 +189,7 @@ stdenv.mkDerivation {
       install -Dm644 desktop/build/libs/Mindustry.jar $out/share/mindustry.jar
       mkdir -p $out/bin
       makeWrapper ${jdk}/bin/java $out/bin/mindustry \
+        ${lib.optionalString stdenv.isLinux "--suffix PATH ${lib.makeBinPath [zenity]}"} \
```

Comment:
--suffix needs a path separator (`:` in this case). see the relevant section in the nixpkgs manual

Reply:
'k!